webui/react/src/hooks/useModal/Experiment/useModalExperimentMove.tsx

```diff
@@ -82,89 +55,73 @@ const moveExperimentWithHandler = async (
 
 const useModalExperimentMove = ({ onClose }: Props): ModalHooks => {
   const canceler = useRef(new AbortController());
-  const { settings: destSettings, updateSettings: updateDestSettings } =
-    useSettings<Settings>(settingsConfig);
+  const [workspaceId, setWorkspaceId] = useState<number>(1);
```

Comment:
fix: because the initial workspaceId is 1 but the Uncategorized workspace is filtered out of the list, the initial state of the modal looks like this, a Workspace named "1" chosen that doesn't exist:
![Screenshot 2022-11-30 at 15-00-12 Experiment 189 - Determined](https://user-images.githubusercontent.com/15078396/204897401-6ed99d9b-5413-40cd-b3ed-c76baf8abcc9.png)

Reply:
You're correct that the Uncategorized workspace has an id of 1. In your screenshot, the 1 in the URL is actually the  **project** id of 1, though that's a minor thing. The problem I was pointing out is that the text in the dropdown was "1", which is not the name of the Uncategorized workspace.